packages/nimble-components/src/table-column/menu-button/cell-view/styles.ts

```diff
@@ -0,0 +1,23 @@
+import { css } from '@microsoft/fast-element';
```

Comment:
Good catch! Interestingly, I can't see that behavior with the original colors I had in the example (I guess they aren't long enough strings). But, I see it with the longer colors, so I guess that should be another copilot win since it's the one that generated all of these colors 😄